app/assets/javascripts/preferences/panes/general-segments/Defaults.tsx

```diff
@@ -0,0 +1,145 @@
+import { Dropdown, DropdownItem } from '@/components/Dropdown';
+import { IconType } from '@/components/Icon';
+import {
+  PreferencesGroup,
+  PreferencesSegment,
+  Subtitle,
+  Text,
+  Title,
+} from '@/preferences/components';
+import { WebApplication } from '@/ui_models/application';
+import {
+  ComponentArea,
+  ComponentMutator,
+  SNComponent,
+} from '@standardnotes/snjs';
+import { FunctionComponent } from 'preact';
+import { useEffect, useState } from 'preact/hooks';
+
+type Props = {
+  application: WebApplication;
+};
+
+const getEditorIconType = (name: string): IconType | null => {
```

Comment:
Hmm, we should go based on editor identifier not name. Names are mutable, but identifiers are generally not.

Reply:
Will make the change. 👍

Reply:
We should make an EditorIdentifier enum:

```typescript
enum EditorIdentifier {
   BoldEditor: 'org.standardnotes.bold-editor',
   PlainEditor: 'plain-editor',
   ...
}
```

Then use that instead of strings directly in the code.